instrumentation/apache-camel-2.20/javaagent/src/test/groovy/io/opentelemetry/javaagent/instrumentation/apachecamel/aws/SnsCamelTest.groovy

```diff
@@ -185,195 +111,259 @@ class S3CamelTest extends AgentInstrumentationSpecification {
             "http.status_code" 200
             "http.url" String
             "net.peer.name" String
-            "net.peer.port" { it == null || Number }
+            "net.peer.port" {it == null || Number}
             "net.transport" "IP.TCP"
           }
         }
       }
-      // camel polling
-      trace(4, 1) {
+      trace(2, 1) {
         span(0) {
-          name "SQS.ReceiveMessage"
+          name "SQS.DeleteMessage"
```

Comment:
Can you use `AwsSpan.sqs()` instead?

Reply:
Yep, that's the idea of a follow-up once I'm sure these changes are heading in the right direction.